scout/adapter/mongo/case.py

```diff
@@ -325,12 +328,37 @@ def update_case(self, case_obj):
                     'research_requested': case_obj.get('research_requested', False),
                 }
             },
-            return_document = pymongo.ReturnDocument.AFTER
+            return_document=pymongo.ReturnDocument.AFTER
         )
 
         LOG.info("Case updated")
         return updated_case
-        ##TODO Add event for updating case?
+
+    def replace_case(self, case_obj):
```

Comment:
I think it is a good idea to have both `replace_case` and `update_case`. Perhaps this might change in the future

Reply:
ok, we'll leave it like that for now then